tests/utils/unit/mockLambdaPayload.ts

```diff
@@ -40,3 +40,18 @@ export const mockLambdaContext = {
   fail: () => {},
   succeed: () => {},
 };
+
+export const mockLambdaScheduledEvent = {
+  "detail-type": "Scheduled Event" as const,
+  id: "event-id",
+  version: "0",
+  account: "123456789012", // Example AWS account ID
```

Comment:
All of these could be just empty string like the one above.

Reply:
Why should this be null?  I want a mock object with some defaults.